CRM/Extension/Browser.php

```diff
@@ -157,21 +157,31 @@ public function getExtensions() {
 
   /**
    * Get a description of a particular extension.
+   * This function is an optimised form of getExtensions() - it returns the
```

Comment:
@mickadoo I think that, with it being a separate function, it probably is clear. The comment is in case someone wants to rewrite it subsequently and is wondering where the old version came from. If you can think of something specific that this comment should contain then we can add that as well.

The part I considered changing was "get a description of a particular extension" as that isn't really what this does - it returns the Extension object. In the end I decided that it was an Optimise PR.